src/config/dynamic-theme-fixes.config

```diff
@@ -1535,6 +1535,13 @@ CSS
 
 ================================
 
+azuresynapse.net
```

Comment:
Is this a internal website?

Reply:
No, it is the portal of the suite for the Microsoft Azure Synapse Analytics cloud offering.

Reply:
Is it only reachable by a subdomain?

Reply:
I think it is yes. Full url is: https://web.azuresynapse.net/